tests/acceptance/users_frontend.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Users Frontend
+  In order to manage users account in the web
+  As a user
+  I need to check user login and registration in joomla! CMS
+
+  Background:
+    Given I see the joomla! Home page
+
+  Scenario: Create user from frontend (index.php?option=com_users)
+    Given I click on the link "Create an account"
+    And I create a user with fields Name "patel", Username "patel", Password "patel" and Email "patel@gmail.com"
+    When I press the "Register"
+    Then I see "Could not instantiate mail function." message
+    And user is created
+
+  Scenario: check the created user in the backend
+    Given There is a user manager page in administrator
+    When I search the user with user name "patel"
+    Then I should see the user "prital"
+
+  Scenario: Login with created user to assure it is blocked
+    Given A newly created user "patel" with password "patel"
+    When He press the "login"
+    Then He should see the "Login denied! Your account has either been blocked or you have not activated it yet." warning
```

Comment:
How is 'newly created' defined? Better say 'not yet activated', because that makes your expectations clear.

Who is 'he'? Use the 'I' phrasing consitently.

```Gherkin
Scenario: User can not login, if the account has not been activated
    Given A not yet activated user with username "patel" and password "patel" exists
    And I am on a frontend page with a login module
    When I enter username "patel" and password "patel" into the login module
    And I press the "login" button
    Then I should see the "Login denied! Your account has either been blocked or you have not activated it yet." warning
```